### PR TITLE
Fix/configure student level

### PR DESCRIPTION
**Pull request by PavloJedi**

### CODE REVIEW

File: back-end/services/MaterialService.js [hunk 1]
```diff
@@ -4,8 +4,12 @@ exports.getAllMaterials = async () => {
   return await MaterialModel.find();
 };
 
-exports.getMaterialsByUnit = async (unitName) => {
-  return await MaterialModel.find({ unit: { $regex: unitName, $options: 'i' } });
+exports.getMaterialsByUnit = async (unitName, level) => {
+  if (level) {
+    return await MaterialModel.find({ unit: { $regex: unitName, $options: 'i' }, level });
+  } else {
```
Comment (andriydzhrvts) [inline back-end/services/MaterialService.js]: Remove else

File: front-end/src/components/RegistrationForm/index.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ import styles from './RegistrationForm.module.scss';
 
 const RegistrationForm = () => {
   const [data, setData] = useState({
-    role: 'student',
+    role: 'Student',
```
Comment (andriydzhrvts) [inline front-end/src/components/RegistrationForm/index.jsx]: Use roles constant

File: back-end/services/MaterialService.js (hunk 1 above)
Comment (PavloJedi) [reply back-end/services/MaterialService.js]: removed else

File: front-end/src/components/RegistrationForm/index.jsx (hunk 2 above)
Comment (PavloJedi) [reply front-end/src/components/RegistrationForm/index.jsx]: used role constant

File: front-end/src/components/RegistrationForm/index.jsx [hunk 3]
```diff
@@ -130,7 +131,7 @@ const RegistrationForm = () => {
 
   const handleChangeActive = (tab) => {
     setActiveTab(tab);
-    setData((prev) => ({ ...prev, role: `${tab === 0 ? 'student' : 'teacher'}` }));
+    setData((prev) => ({ ...prev, role: `${tab === 0 ? 'Student' : 'Teacher'}` }));
```
Comment (andriydzhrvts) [inline front-end/src/components/RegistrationForm/index.jsx]: use student role constant

Comment (PavloJedi) [reply front-end/src/components/RegistrationForm/index.jsx]: @andriydzhrvts updated to role constants

**Review by andriydzhrvts — APPROVED** [review]

**Pull request merged**